src/ui/public/config/config_provider.js

```diff
@@ -0,0 +1,7 @@
+export function ConfigProvider(config) {
+  return {
+    getConfig: (key, defaultValue) => {
```

Comment:
What value do you see in this service, over just defining `const getConfig = (...args) => config.get(...args)` above where it's used?